tests/all/unit/api/helpers/test_export_helpers.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+
+from app import current_app as app
+from tests.all.integration.utils import OpenEventTestCase
+from tests.all.integration.setup_database import Setup
```

Comment:
This is unit test package, and you are importing integation test utils and setup_database. You have not written a unit test here. It's still an integration test

Reply:
Think yourself, why do I need to setup app context and DB to test ordering of list an dictionary?

Reply:
I guess the test is now correct, please review.